sdv/metadata/multi_table.py

```diff
@@ -473,15 +480,15 @@ def add_table(self, table_name):
                 The name of the table to add to the metadata.
 
         Raises:
-            Raises ``ValueError`` if ``table_name`` is not valid.
+            Raises ``InvalidMetadataError`` if ``table_name`` is not valid.
         """
         if not isinstance(table_name, str) or table_name == '':
-            raise ValueError(
+            raise InvalidMetadataError(
```

Comment:
@amontanez24 The issue only mentioned the `validate` methods. I'm not sure if these should be changed as well?

Reply:
Yeah we might as well. If it has to do with a bad input to the metadata I think we should change it for consistency